bonsai/examples/governance/script/Deploy.s.sol

```diff
@@ -75,7 +75,7 @@ contract Deploy is Script, BonsaiCheats {
         } else {
             // Deploy an IRiscZeroVerifier contract instance. Relay is stateless and owner-less.
             IRiscZeroVerifier verifier;
-            address verifierAddr = vm.envOr("DEPLOY_VERFIER_ADDRESS", address(0));
+            address verifierAddr = vm.envOr("DEPLOY_VERIFIER_ADDRESS", address(0));
```

Comment:
It is important to call out that there are typo fixes that could lead to compatibility issues.

Here is the Bonsai governance Solidity script. It uses an environment variable called "DEPLOY_VERFIER_ADDRESS". Fixing this typo may cause issues with the upstream.